azure-pipelines.yml

```diff
@@ -14,12 +14,12 @@ trigger:
 
```

Comment:
In the Azure pipeline, it isn't clear to me why line 66 (`pytest --pyargs hyperspy`) does not work if one simply uncomments it.
From the current setup files, it seems that the `dev` dependencies (e.g. pytest, coverage...) are not installed.

Maybe it should be clarified in the README file.